connectors/src/connectors/microsoft/temporal/activities.ts

```diff
@@ -625,11 +625,13 @@ function sortForIncrementalUpdate(changedList: DriveItem[], rootId: string) {
 
   const sortedItemList = changedList.filter((item) => {
     if (item.id === rootId) {
+      // Found selected root
       return true;
     }
 
-    if (!item.parentReference) {
-      return false;
+    if (!item.parentReference || !item.parentReference.id) {
+      // Root folder of the drive, always include it
```

Comment:
1. I would keep the !parentReference => return false as before, but if we want to have the root folder either rootId should be the root folder id so the above condition would match, or we do (item.id === rootId || item.root)` above ; testing item.root is the clean way of knowing if we're on the root folder of a drive afaict

Reply:
2. are we sure we always want to include the root folder of the drive? this will result in not filtering the list at all, since all elements in the change list are descendants of the root folder; while in many cases we want to exclude elements of the changelist that are not in a toplevel folder